mitosheet/src/jupyter/api.tsx

```diff
@@ -508,6 +510,14 @@ export default class MitoAPI {
         })
     }
 
+    async getMergeParams(
+        destinationSheetIndex: number
+    ): Promise<MergeParams | undefined> {
+        return await this.getParams('merge', undefined, {
+            'destination_sheet_index': destinationSheetIndex
```

Comment:
Ah -- this is why we need it in the execution data! I see. Nice. Maybe leave a comment in pivot.py and merge.py so we can remember in the future.